www/webapp/templates/components/priceTable/priceTable.html

```diff
@@ -7,18 +7,58 @@
         <div class="card-header flex-fill mb-2">
           <h4 class="my-0 ">{{ subscription_type.name }}</h4>
         </div>
+
         <div class="card-body">
+          <input type="hidden" class="subscription-type-id" value="{{ subscription_type.id }}">
+          <!-- <div class="subscription-type-id d-none">
+            {{ subscription_type.id }}
+          </div> -->
             <h1 class="card-title pricing-card-title py-3">
               ${{ "{:.2f}".format(subscription_type.price) }} 
               {{ subscription_type.currency }}
           </h1>
           
-          <button type="button" class="w-100 btn btn-lg btn-primary">Subscribe NOW!</button>
+          <button type="button" class="w-100 btn btn-lg btn-subscribe btn-outline-primary">Select Subscribe!</button>
         </div>
       </div>
     </div>
     {% endfor %}
   </div>
 </main>
+<script>
+
+let is_redirect_price_table = "{{ is_redirect_price_table }}";
```

Comment:
I don't think processing redirect on server side is needed plaase

```suggestion
const urlParams = new URLSearchParams(window.location.search);
const myParam = urlParams.get('redirect_url');
```